pybamm/version

```diff
@@ -1 +1 @@
-21, 8
+21, 9
```

Comment:
Or maybe this should be 21, 09

Reply:
I think it is read as an `int` and the formatted when it is printed. About PyPi, I think I saw some warning in the actions (probably the one that builds the wheels) that said that PyPi would ignore the 0, so we might as well remove it...